undefined_syms.txt

```diff
@@ -390,4 +390,7 @@ D_06009244 = 0x06009244;
 D_060041F4 = 0x060041F4;
 D_0600A054 = 0x0600A054;
 D_06008688 = 0x06008688;
-D_04049210 = 0x04049210;
\ No newline at end of file
+D_04049210 = 0x04049210;
+
+// z_bg_hidan_firewall
+D_0600DA80 = 0x0600DA80;
```

Comment:
theres a merge conflict here in undefined syms but youll catch that when you merge master